src/index.js

```diff
@@ -120,7 +120,12 @@ class ServerlessAppSyncSimulator {
       const sub = {
         expression: [
           'anyof',
-          ...this.options.watch.map((glob) => ['match', glob]),
+          ...this.options.watch.map((glob) => {
+            if (Array.isArray(glob)) {
+              return glob;
+            }
+            return ['match', glob];
```

Comment:
`glob` is expected to be string if not array, and this line is added for backward compatibility.  Not validating types other than above in this PR.